projects/ui/src/Components/Common/Banner/BannerHeading.tsx

```diff
@@ -24,6 +25,16 @@ export function BannerHeading({
 }) {
   const { pageContentIsWide } = useContext(AppContext);
 
+  const bgImage = useMemo(() => {
+    if (!!bgImageURL) {
+      return bgImageURL;
+    }
+    if (!!bannerImageURL) {
+      return bannerImageURL;
```

Comment:
Does React react to a change in a file that is imported?  I'd believe it, but it doesn't seem intuitive

Reply:
Sorry, I see this is actually importing a variable from that file.   Still, it feels like importing it would break the normal tracking.   

Again, totally believe it, but it feels very weird.

Reply:
I'm fairly confident that this would result in the imports (both the `Banner`, and the `bannerImageURL` - which definitely could be named better) not causing any additional re-renders after the first render. So it would be the same as if the deps array was: `[bgImageURL]`.

However, to be clearer and to not run into any React weirdness, I'll update it to just use `[bgImageURL]`.